src/Query/ResultSetMapping.php

```diff
@@ -552,25 +551,4 @@ public function addMetaResult(
 
         return $this;
     }
-
-    public function addNewObjectAsArgument(string|int $alias, string|int $objOwner, int $objOwnerIdx): static
-    {
-        $owner = [
-            'ownerIndex' => $objOwner,
-            'argIndex' => $objOwnerIdx,
-        ];
-
-        if (! isset($this->nestedNewObjectArguments[$owner['ownerIndex']])) {
-            $this->nestedNewObjectArguments[$alias] = $owner;
-
-            return $this;
-        }
-
-        $this->nestedNewObjectArguments = array_merge(
-            [$alias => $owner],
-            $this->nestedNewObjectArguments,
-        );
-
-        return $this;
-    }
```

Comment:
This looks a lot like a breaking change.

Reply:
This function was created in my previous PR, so not very break

Reply:
Indeed, we haven't issued a release yet that includes this method. The change is fine.